core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.ts

```diff
@@ -131,6 +132,14 @@ export class DotDeviceSelectorSeoComponent implements OnInit {
         this.overlayPanel.hide();
     }
 
+    /**
+     * Emit selected changes
+     * @param DotDevice device
+     */
+    changeSeoMediaEvent(tile: string) {
+        this.changeSeoMedia.emit(tile);
+    }
```

Comment:
Typo?

```suggestion
    changeSeoMediaEvent(title: string) {
        this.changeSeoMedia.emit(title);
    }
```

Reply:
@rjvelazco It is Social Media Tiles that's why I called tile

Reply:
Thanks @manuelrojas TIL!